ihatemoney/forms.py

```diff
@@ -31,14 +31,14 @@ def strip_filter(string):
         return string
 
 
-def get_billform_for(project, set_default=True, **kwargs):
+def get_billform_for(project, set_default=True, bill_type=BillTypes.loan.value, **kwargs):
```

Comment:
You probably just don't need to add bill_type here, let it go trough the kwargs, and set up a default value in the Form's ``__init__`` method.